tools/scripts/build_gz.sh

```diff
@@ -42,6 +42,19 @@ if [[ ! -z "$4" && "$4" != "n" ]]; then
   libName=`echo "$2" | grep -oP "(?<=gz-).*"`
   libName="${libName//-/_}"
 
+  majorVersion="${version/\.*/}"
+  # Make sure the majorVersion is a valid number
+  numberCheckRegex='^[0-9]+$'
+  if [[ $majorVersion =~ $numberCheckRegex ]]; then
+    # If this is ign-gazebo (gz-sim < 6), the upload_doc.sh will upload to api/gazebo so we'll need to
+    # sync to api/sim manually
+    if [[ "$libName" == "sim" && "$majorVersion" -le 6 ]]; then
```

Comment:
Should this be `-lt 6`?

Reply:
I updated the comment. It should be <= 6. so `-le 6` is correct.